### ISSUE TITLE
Warning: Failed prop type: Invalid prop `children` supplied to `TableView` RN44

### ISSUE THREAD

**Issue by sibelius** [Purii/react-native-tableview-simple#33]
this structure:

```jsx
<TableView>
   <Section>...</Section>
   {showMore && (<Section>...</Section>}
</TableView>
```

generate the following warning:

Failed prop type: Invalid prop `children` supplied to `TableView`

related to:
- https://github.com/Purii/react-native-tableview-simple/issues/18
- https://github.com/Purii/react-native-tableview-simple/issues/11
- https://github.com/Purii/react-native-tableview-simple/issues/10

**Comment by sibelius** [comment]
can we just simple remove this check https://github.com/Purii/react-native-tableview-simple/blob/master/components/TableView.js#L20?

**Comment by austenmc** [comment]
Running into this as well. Seems like a safe fix.

**Comment by Purii** [comment]
That would be the easiest way. But I like to use propTypes everywhere and do not want to declare use exceptions, if that could be avoided. I will look after it 👍

**Comment by austenmc** [comment]
You could build a custom validator that checks that children are elements or arrays of elements.

https://facebook.github.io/react/docs/typechecking-with-proptypes.html

**Comment by Purii** [comment]
> false, null, undefined, and true are valid children.

(https://facebook.github.io/react/docs/jsx-in-depth.html)

Could you please check, if your issues are fixed with `v0.16.5`?
